patches/video.js+7.13.3.patch

```diff
@@ -1,8 +1,33 @@
 diff --git a/node_modules/video.js/dist/video.es.js b/node_modules/video.js/dist/video.es.js
-index a07aa77..3f7bc30 100644
+index a07aa77..142743f 100644
 --- a/node_modules/video.js/dist/video.es.js
 +++ b/node_modules/video.js/dist/video.es.js
-@@ -13959,7 +13959,7 @@ var ProgressControl = /*#__PURE__*/function (_Component) {
+@@ -13107,12 +13107,12 @@ var TimeTooltip = /*#__PURE__*/function (_Component) {
+     // `SeekBar` before hitting the `seekBarPoint`
+ 
+ 
+-    var spaceLeftOfPoint = seekBarRect.left - playerRect.left + seekBarPointPx; // This is the space right of the `seekBarPoint` available within the bounds
++    var spaceLeftOfPoint = seekBarRect.left - playerRect.left + seekBarPointPx - 24; // This is the space right of the `seekBarPoint` available within the bounds
+     // of the player. We calculate the number of pixels from the `seekBarPoint`
+     // to the right edge of the `SeekBar` and add to that any gap between the
+     // right edge of the `SeekBar` and the player.
+ 
+-    var spaceRightOfPoint = seekBarRect.width - seekBarPointPx + (playerRect.right - seekBarRect.right); // This is the number of pixels by which the tooltip will need to be pulled
++    var spaceRightOfPoint = seekBarRect.width - seekBarPointPx + (playerRect.right - seekBarRect.right) - 24; // This is the number of pixels by which the tooltip will need to be pulled
+     // further to the right to center it over the `seekBarPoint`.
+ 
+     var pullTooltipBy = tooltipRect.width / 2; // Adjust the `pullTooltipBy` distance to the left or right depending on
+@@ -13349,7 +13349,8 @@ var MouseTimeDisplay = /*#__PURE__*/function (_Component) {
+ 
+     var time = seekBarPoint * this.player_.duration();
+     this.getChild('timeTooltip').updateTime(seekBarRect, seekBarPoint, time, function () {
+-      _this2.el_.style.left = seekBarRect.width * seekBarPoint + "px";
++      _this2.el_.style.right = seekBarRect.width * (1 -seekBarPoint) + "px";
++      _this2.el_.style.width = seekBarRect.width * seekBarPoint + "px";
```

Comment:
this will increase the size of the mouse time display(small 1px vertical line on timeline bar which is visible when you moving the cursor) 
Before
![image](https://user-images.githubusercontent.com/51168865/126442841-356e29d9-ae42-4579-99ec-be6378a95616.png)
Now
![image](https://user-images.githubusercontent.com/51168865/126442337-a389dc31-5522-4cee-a57c-864592d09ace.png)

Design
![image](https://user-images.githubusercontent.com/51168865/126442749-3eb9a788-1d8f-428f-a08b-e729da00f05a.png)